shell.nix

```
@@ -26,5 +26,8 @@ pkgs.mkShellNoCC {
     # Helper to review Nixpkgs PRs
     # See CONTRIBUTING.md
     nixpkgs-review
+    # Linter and Formatter for shell scripts
+    shellcheck
```

`shellcheck-minimal` is the ShellCheck derivation `writeShellApplication` uses, which is supposedly easier to bootstrap.

```suggestion
    shellcheck-minimal
```